app/assets/stylesheets/hyrax/_collections.scss

```diff
@@ -258,3 +242,231 @@ button.branding-banner-remove:hover {
     }
   }
 }
+
+// From coll-landing branch -- TODO: Confirm these are still needed.
+.parent{
```

Comment:
This seems very generic.  Are we certain there won't be a parent class anywhere else in the app? (same with `.inner`)

Reply:
I agree with you, but I didn't create those style rules so I left them as-is for the time being.

Reply:
✅
I found no evidence that these CSS classes (.parent & .inner) were being used so I deleted their rules.